src/server/home/index.njk

```diff
@@ -1,56 +1,48 @@
 {% extends 'layouts/page.njk' %}
 
 {% block content %}
-  {{ appHeading({
-    text: heading,
```

Comment:
We could do with updating the heading here too - to "Move animals under disease control restriction"?